gointerfaces/remote/ethbackend.pb.go

```diff
@@ -567,8 +559,8 @@ type EngineForkChoiceUpdatedRequest struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	HeadBlockHash *EnginePreparePayload    `protobuf:"bytes,1,opt,name=headBlockHash,proto3" json:"headBlockHash,omitempty"`
-	SafeBlockHash *EngineForkChoiceUpdated `protobuf:"bytes,2,opt,name=safeBlockHash,proto3" json:"safeBlockHash,omitempty"`
+	Forkchoice *EngineForkChoiceUpdated `protobuf:"bytes,2,opt,name=forkchoice,proto3" json:"forkchoice,omitempty"`
```

Comment:
Hmm. It looks like you didn't regenerate this file to match .proto.  Forkchoice should have id 1 and Prepare should have id 2.

Reply:
Will open a new PR with a fix to this